test-data/unit/check-async-await.test

```diff
@@ -184,7 +184,7 @@ async def f() -> None:
 [typing fixtures/typing-full.pyi]
 
 [case testAsyncForComprehension]
-# flags: --fast-parser --python-version 3.6
+# flags: --fast-parser --python-version 3.6 --no-site-packages
```

Comment:
It's unfortunate you had to add `--no-site-packages` to so many test files. Maybe it's better to set some more conservative default options for all data-driven testcases in one place?

Reply:
I think I can add it somewhere in `testcheck.py`, I'll take a look.